docs/java-rest/high-level/security/grant-api-key.asciidoc

```diff
@@ -0,0 +1,44 @@
+--
+:api: grant-api-key
+:request: GrantApiKeyRequest
+:response: GrantApiKeyResponse
```

Comment:
```suggestion
:response: CreateApiKeyResponse
```

Because Grant API Key is really just a special way to call Create API Key, it actually uses the `CreateApiKeyResponse`